companion-generator/src/main/java/com/gradle/cucumber/companion/generator/CompanionGenerator.java

```diff
@@ -27,6 +27,8 @@
 
 public class CompanionGenerator {
 
+    private static final String NEW_LINE = "\n";
```

Comment:
❔Is your idea to simply guarantee that all files use the Unix-line encoding, because they might get executed on TD agents?